services/blockchain-indexer/shared/indexer/blockchainIndex.js

```diff
@@ -111,24 +119,19 @@ const getTransactionExecutionStatus = (tx, events) => {
 	return txExecResultEvent.data.data === '0801' ? 'success' : 'fail';
 };
 
-// TODO: Move this require to top
-// const { KV_STORE_KEY } = require('../constants');
-// const EVENT_NAME = Object.freeze({
-// 	LOCKED: 'locked',
-// 	UNLOCKED: 'unlocked',
-// });
-
-// const updateTotalLockedAmounts = (tokenIDLockedAmountChangeMap, dbTrx) => BluebirdPromise.map(
-// 	Object.entries(tokenIDLockedAmountChangeMap),
-// 	async ([tokenID, lockedAmountChange]) => {
-// 		const tokenKey = KV_STORE_KEY.PREFIX.TOTAL_LOCKED.concat(tokenID);
-// 		const curLockedAmount = BigInt(await keyValueTable.get(tokenKey) || 0);
-// 		const newLockedAmount = curLockedAmount + lockedAmountChange;
-
-// 		await keyValueTable.set(tokenKey, newLockedAmount, dbTrx);
-// 	},
-// 	{ concurrency: Object.entries(tokenIDLockedAmountChangeMap).length },
-// );
+const updateTotalLockedAmounts = (tokenIDLockedAmountChangeMap, dbTrx) => BluebirdPromise.map(
+	Object.entries(tokenIDLockedAmountChangeMap),
+	async ([tokenID, lockedAmountChange]) => {
+		if (lockedAmountChange === BigInt(0)) return;
+
+		const tokenKey = KV_STORE_KEY.PREFIX.TOTAL_LOCKED.concat(tokenID);
```

Comment:
Please set the value to `0` instead. Else, when querying, the response will be undefined/null.
```suggestion
	async ([tokenID, lockedAmountChange]) => {
		const tokenKey = KV_STORE_KEY.PREFIX.TOTAL_LOCKED.concat(tokenID);
```

Reply:
Applied.